site/content/kapp-controller/docs/develop/packaging-tutorial.md

```diff
@@ -387,6 +387,13 @@ Once the simple-package-repository has a "**Reconcile succeeded**" description,
 we're ready to continue! You can exit the watch by hitting control-c or
 clicking: `^C`
 
+
+Alternatively, we can add the PackageRepository using `kctrl` without authoring the YAML for it.
```

Comment:
for these files they're generated from a katacoda tutorial as indicated at the top of the file.  right now the katacoda tutorial is the definitive source and it's auto-edited by a script into this markdown tutorial. I realize maybe it would've been less confusing for maintenance if the directionality were reversed so maybe feel free if you want to look into that, or we could let the contents diverge.   the thing i appreciate about the katacoda tutorial is that it makes it very easy to ensure that there's no typos in the instructions, since you can click through the tutorial to run the code.

Reply:
see https://github.com/vmware-tanzu/carvel/tree/develop/tutorials